config/application.rb

```diff
@@ -84,7 +84,7 @@ class Application < Rails::Application
     # parameters by using an attr_accessible or attr_protected declaration.
     #config.active_record.whitelist_attributes = true
 
-    config.autoload_paths += %W(#{config.root}/lib)
+    config.autoload_paths += %W(#{config.root}/lib #{config.root}/app)
```

Comment:
This isn't required. Directories within the app dir are automatically autoloaded. You might need to restart the app and/or stop Spring first `spring stop`.

Reply:
Seeing the same issue @jollopre has reported here. Fine to keep this in when needed.